docs/2.8.0/docs/query/pqs-user-guide.rst

```diff
@@ -835,8 +895,140 @@ The following schema is representative for the exported ledger data. It is subje
+
+Active contracts are preserved under a new offset, while all other
+transaction-related data up to, and including the target offset is
+deleted.
+
+The target offset, ie. the offset provided via ``--prune-target`` or as
+argument to ``prune_to_offset``, is the transaction with the highest
+offset that will be deleted by the pruning operation.
+
+Note: If the provided offset (i.e. via ``--prune-target``, or as
+argument to ``prune_to_offset``) does not have a transaction record,
+then the effective target offset will be the oldest transaction offset
+that succeeds (is greater than) the provided offset.
+
+When using either pruning method, the following data will be changed:
+
+-  The offset of active contracts will be moved to the oldest known
+   offset which succeeds the pruning target offset, i.e. the offset of
+   the oldest transaction that is unaffected by the pruning operation.
+
+The following data will be deleted:
+
+-  Transactions with offsets up to and including the target offset.
+-  Events, archived contracts and exercise payloads associated with the
+   deleted transactions.
+
+The following data will be unaffected: 
+
+- Transaction related data (event, choices, or contracts) for transaction with
+  an offset that is greater than the effective pruning target offset.
+
+Pruning is a destructive operation and cannot be undone. If necessary,
+make sure to backup your data before performing any pruning operations.
+
+There are some constraints when using either pruning method:
+
+1. The provided target offset must be within the bounds of the
+   contiguous history. If the target offset is outside the bounds, an
+   error will be raised.
+2. The pruning operation cannot coincide with the latest consistent
+   checkpoint of the contiguous history. If it does, an error will be
+   raised.
+
+Pruning with PQS CLI
+-----------------------
+
+The PQS CLI provides a ``prune`` command that allows you to prune the
+ledger data up to a specified offset, timestamp, or duration.
+
+For detailed information on all available options, please run
+``./PQS.jar datastore postgres-document prune --help-verbose``.
+
+To use the ``prune`` command, you need to provide a pruning target as an
+argument. The pruning target can be an offset, a timestamp (ISO 8601),
+or a duration (ISO 8601):
+
+::
+
+   ./PQS.jar datastore postgres-document prune --prune-target <pruning_target>
+
+By default, the ``prune`` command performs a dry run, which means it
+will only display the effects of the pruning operation without actually
+deleting any data. To execute the pruning operation, you need to add the
+``--prune-mode Force`` option:
+
+::
+
+   ./PQS.jar datastore postgres-document prune --prune-target <pruning_target> --prune-mode Force
+
+In addition to providing an offset as ``--prune-target``, a timestamp
+or duration can also be used as pruning cut-off. For example, to prune
+data older than 30 days (relative to now), you can use the following
+command:
```

Comment:
```suggestion
Instead of providing an offset as the ``--prune-target``, you can use a timestamp
or duration as the pruning cutoff. For example, the following command prunes
data older than 30 days (relative to now):
```